src/app/system/layout.tsx

```diff
@@ -16,7 +16,8 @@ const OPTIONS: {
         { name: 'Register Admin', link: '/system/new/admin' },
     ],
     [Role.ADMIN]: [
-        { name: 'create Promotor', link: '/system/new/promoter' },
+        { name: 'Create Room', link: '/system/new/room' },
```

Comment:
no existe ningun room

Reply:
removed in [bb130a3](https://github.com/mael098/appcine/pull/26/commits/bb130a36a79195e9063f4c7a85b672fb72efc0d4)